.gitignore

```diff
@@ -12,4 +12,6 @@ dist/scripts/*.php
 package-lock.json
 composer.lock
 coverage
-.idea
\ No newline at end of file
+.idea
+dist/*.map
+dist/*.php
```

Comment:
This actually should be allowed, because when it comes to blocks, this is what loads dependencies.